R/utils.R

```diff
@@ -1,3 +1,48 @@
+register_functions_to_cluster <- function(cluster) {
```

Comment:
roxygen import statement can be added for the utils

Reply:
It was actually there, just got pushed under the `register_functions_to_cluster` function. Fixed in [5a36833](https://github.com/RECETOX/recetox-aplcms/pull/154/commits/5a36833b0e22bbf440320b814a0d088db1d19a32).